pkg/syslog/rfc5424/parser.go

```diff
@@ -19,7 +19,7 @@ func Parse(buf []byte) (*Message, error) {
 	}
 
 	if cursor < len(buf) {
-		msg.Msg = buf[cursor:]
+		msg.Msg = buf[cursor+1:]
```

Comment:
I think for consistency w/ the other parse funcs, this should actually be fixed inside `parseStructuredData()` by changing [this line](https://github.com/benburkert/flynn/blob/logging-fix-parser/pkg/syslog/rfc5424/parser.go#L155) to a `*cursor += 2`. The other parse methods skip over the trailing space (like [this one](https://github.com/benburkert/flynn/blob/d51e5120de0838e2f8d0f91cb71eda6907bb70a3/pkg/syslog/rfc5424/parser.go#L106))